gradle/javadoc.gradle

```diff
@@ -14,8 +14,11 @@ task mockitoJavadoc(type: Javadoc) {
     title = "Mockito ${project.version} API"
     def javadocJdk6Hack = "jdk6-project-version-insert.min.js"
 
-    options.doclet "org.mockito.javadoc.JavadocExclude"
-    options.docletpath = [rootProject.file('./buildSrc/build/classes/main/')]
+    // The doclet is more flexible but still allow to generate internal links, while exclude is more of a hammer.
+    exclude 'org/mockito/internal'
```

Comment:
This change does not work, see https://github.com/mockito/mockito/pull/760#issuecomment-261229080